pint/models/spindown.py

```diff
@@ -25,29 +25,28 @@ def __init__(self):
         # The number of terms in the taylor exapansion of spin freq (F0...FN)
         self.num_spin_terms = maxderivs
 
```

Comment:
I don't think there is a single identifiable number like that. It depends on the tim file, which we do not have access to here. There are two cases that could cause problems:

1) The number of TOAs is lower than the number of timing model parameters
2) The condition number of the design matrix becomes too large

Number 1 is solved by using appropriate priors. Number two is solved by the fitter, by normalizing the design matrix when creating inversion matrices. I have done both of these before. IMO, setting a number of maxderivs is ennecessarily limiting, even though in practice it would (almost) always make sense.